src/sass/base/_typography.scss

```diff
@@ -1,3 +1,173 @@
 // -----------------------------------------------------------------------------
 // This file contains default typographic styles for copy text.
 // -----------------------------------------------------------------------------
+
+// From Typebase.scss v0.1.0 | https://github.com/devinhunt/typebase.css/blob/master/src/typebase.scss
+
+// Typesetting variables.
```

Comment:
Are we going to keep variables compartmentalized within components or have a central variables file? My thinking; in a component, pull in the centralized value if it exists, otherwise revert to a default set in the component. Thoughts?

Reply:
Yes, this is what I was thinking. After having used the 'one file to rule them all' method, I appreciate grouping my variables with where they're in use.

Reply:
I agree to pull in from centralized value (will this be a centralized file then?) and then have default set in component as well.